warehouse/ingest-core/src/main/java/datawave/ingest/data/tokenize/StandardTokenizer.java

```diff
@@ -18,12 +18,12 @@
  */
 
 import java.io.IOException;
```

Comment:
Much of this file is derived from a similar class in Lucene's codebase as such the updates here mirror that class' changes.